.storybook/decorators/FillHeight.tsx

```diff
@@ -1,10 +1,10 @@
 import React from 'react';
 import type { Story } from '@storybook/react';
 
-function FillHeight(VisCanvasStory: Story) {
+function FillHeight(WrappedStory: Story) {
```

Comment:
`aStory` ?

Reply:
Needs to have a capital letter because it's a component.

Reply:
`MyStory` ? It is just that I don't like `WrappedStory` (wrapped by whom ? by what ?)

Reply:
It's what they use for the HOC pattern: https://reactjs.org/docs/higher-order-components.html, but I hesitated with `MyStory`, so I'm fine with that too 👍